src/components/Accordion/AccordionItem.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Accordion.css';
+
+class AccordionItem extends React.Component {
+  constructor(props) {
+    super(props);
+    this.accordionItemToggle = this.accordionItemToggle.bind(this);
+    this.state = {
+      hidden: true,
+    };
+  }
+  accordionItemToggle() {
+    const currentState = this.state.hidden;
+    this.setState({ hidden: !currentState });
+  }
+  render() {
+    return (
+      <li
+        className="p-accordion__group"
```

Comment:
Don't think you need to wrap this `className` onto a new line. It makes the JSX look weird.